agent/js/src/wd_server.js

```diff
@@ -155,7 +155,8 @@ WebDriverServer.prototype.initIpc = function() {
  */
 WebDriverServer.prototype.init = function(args) {
   'use strict';
-  args.startURL = args.task['Capture Video'] ? ORANGE_PAGE_URL_ : BLANK_PAGE_URL_;
+  args.startURL = args.task['Capture Video'] && !args.task.lighthouse ? 
+      ORANGE_PAGE_URL_ : BLANK_PAGE_URL_;

```

Comment:
Discussed separately, but conclusion is that this will not be needed for now. 

I will investigate potential IPC communication in lighthouse to notify the agent that the orange screen is being displayed and will fallback to trimming the video up until lighthouse's orange page rather than WPTs.